src/components/LocationPage/ShareButtons.js

```diff
@@ -41,17 +42,58 @@ const InnerContent = props => {
     : imageBaseUrl +
       `states/${stateId.toLowerCase()}/chart/${chartIdentifier}/export.png`;
 
+  const downloadDate = moment().format('YYYY-MM-DD');
+
+  function makeDownloadFilename(chartIdentifier) {
+    const chartDownloadType = {
+      0: 'infection_growth',
```

Comment:
Can you use the actual metric values (e.g. `Metric.INFECTION_GROWTH`) instead of raw numbers?  If somebody were to e.g. reorder the enum, then this would break.

Reply:
ah yes true